apps/blockchain/test/blockchain/state_test.exs

```diff
@@ -186,6 +171,31 @@ defmodule Blockchain.StateTest do
     |> is_nil()
   end
 
+  def configuration(hardfork) do
```

Comment:
maybe we can reuse the method from https://github.com/poanetwork/mana/blob/master/apps/blockchain/lib/blockchain/chain.ex#L163 ?